pkg/render/logstorage.go

```diff
@@ -611,15 +660,36 @@ func (es elasticsearchComponent) nodeSets() []esv1.NodeSet {
 func (es elasticsearchComponent) nodeSetTemplate() esv1.NodeSet {
 	pvcTemplate := es.pvcTemplate()
 
+	config := map[string]interface{}{
+		"node.master":                 "true",
+		"node.data":                   "true",
+		"node.ingest":                 "true",
+		"cluster.max_shards_per_node": 10000,
+	}
+
+	switch es.authentication.(type) {
+	case OIDCAuthentication:
+		oidcAuth := es.authentication.(OIDCAuthentication)
+		config["xpack.security.authc.realms.oidc.oidc1"] = map[string]interface{}{
+			"order":                       1,
+			"rp.client_id":                oidcAuth.ClientID,
+			"rp.response_type":            "code",
+			"rp.redirect_uri":             fmt.Sprintf("https://%s:9443/tigera-kibana/api/security/v1/oidc", oidcAuth.SiteURL),
+			"rp.requested_scopes":         oidcAuth.RequestedScopes,
+			"op.issuer":                   oidcAuth.IssuerURL,
+			"op.authorization_endpoint":   oidcAuth.AuthorizationEndpoint,
+			"op.token_endpoint":           oidcAuth.TokenEndpoint,
+			"op.jwkset_path":              oidcAuth.JWKSetURI,
+			"rp.post_logout_redirect_uri": fmt.Sprintf("https://%s:9443/tigera-kibana/logged_out", oidcAuth.SiteURL),
+			"claims.principal":            oidcAuth.UsernameClaim,
+			"claims.group":                oidcAuth.GroupsClaim,
```

Comment:
ES supports a list of groups right?

Reply:
Yes, but I think it's provided as a comma separated or space separate list.

Reply:
I'll test this out when I'm back from vacation, I think it would be OK to leave it for the moment if you're cool with it